src/components/themed/WalletListCurrencyRow.js

```diff
@@ -38,166 +24,56 @@ type Props = {
-    return `${symbol != null ? symbol + ' ' : ''}${finalCryptoAmount}`
-  } catch (error) {
-    if (error.message === 'Cannot operate on base16 float values') {
-      const errorMessage = `${error.message}: Currency code - ${currencyCode}, balance - ${balance}, demonination multiplier: ${multiplier}`
-      console.error(errorMessage)
-    }
-    console.error(error)
-  }
-
-  return ''
-}
-
-export const getDifference = (getRateParams: GetDifferenceParams) => {
-  const { currencyCode, exchangeRate, exchangeRates, fiatExchangeRate = '0' } = getRateParams
-  // Create the default empty result function
-  const result = (value, style = 'neutral') => ({
-    differencePercentageString: value != null ? `${value}%` : '',
-    differencePercentageStyle: style
-  })
-  // Yesterdays Exchange Rate
-  const currencyPair = `${currencyCode}_iso:USD_${getYesterdayDateRoundDownHour()}`
-  const yesterdayUsdExchangeRate = toFixed(exchangeRates[currencyPair] ?? '0', 0, 2)
-  const yesterdayExchangeRate = mul(yesterdayUsdExchangeRate, fiatExchangeRate)
-  // Return the Exchange Rate without `percentageString` in case we are missing yesterday's rate
-  if (zeroString(yesterdayExchangeRate)) return result()
-  // Calculate the percentage difference in rate between yesterday and today
-  const differenceYesterday = sub(exchangeRate, yesterdayExchangeRate)
-  const differencePercentage = mul(div(differenceYesterday, yesterdayExchangeRate, 3), '100')
-  // Return zero result
-  if (zeroString(differencePercentage)) return result('0.00')
-  // If not zero, create the `percentageString`
-  const percentageString = abs(differencePercentage)
-  // Return Positive result if greater then zero
-  if (gt(differencePercentage, '0')) return result(`+${percentageString}`, 'positive')
-  // If it's not zero or positive, it must be a Negative result
-  return result(`-${percentageString}`, 'negative')
-}
-
+/**
+ * Renders a WalletListRow with the hide-able balance children.
+ */
 export const WalletListCurrencyRowComponent = (props: Props) => {
   const { currencyCode, showRate = false, onPress, onLongPress, gradient, walletId, walletName, tokenCode } = props
   const dispatch = useDispatch()
   const theme = useTheme()
   const styles = getStyles(theme)
 
-  const edgeWallet = useSelector(state => state.core.account.currencyWallets[walletId])
-  const exchangeRates = useSelector(state => state.exchangeRates)
+  const currencyInfo = useSelector(state => state.core.account.currencyWallets[walletId].currencyInfo)
+  const fiatCurrencyCode = useSelector(state => state.core.account.currencyWallets[walletId].fiatCurrencyCode)
+  const balance = useSelector(state => state.core.account.currencyWallets[walletId].balances[currencyCode] ?? '0')
+  const name = useSelector(state => state.core.account.currencyWallets[walletId].name)
```

Comment:
Incorrect selector usage again. The way it was before was more correct, since it included the necessary subscriptions.